src/clientes.hs

```diff
@@ -1,8 +1,8 @@
 import Cliente
 
-rodri       = Cliente "Rodri" 55 [] []
-marcos      = Cliente "Marcos" 40 [rodri] []
-cristian    = Cliente "Cristian" 2 [] []
+rodri       = Cliente "Rodri" 55 [] [Tintico]
```

Comment:
Por qué no usar la función de tomarTrago?

Reply:
That would be nice

Reply:
Y crear una nueva que sea rodriTomado? Igual no afecta (as far as I know) que esté ahí o no el trago

Reply:
Dale, lo de los saltos de línea ya quedó :+1:

Reply:
Pero le hago esa función

Reply:
Como quieras, te comentaba que lo otro de los saltos de línea ya estaba :see_no_evil: 

**[ojota]** Si haces la función `rodriTomado` que sea el resultado de `tomarTrago [<tragos>] rodri`, fijate también de reemplazar las aparciones de `rodri` por `rodriTomado`